hal/targets/hal/TARGET_Freescale/TARGET_KSDK2_MCUS/TARGET_K64F/entropy_hardware_poll.c

```diff
@@ -0,0 +1,81 @@
+/*
+ *  Hardware entropy collector for the K64F, using Freescale's RNGA
+ *
+ *  Copyright (C) 2006-2015, ARM Limited, All Rights Reserved
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may
+ *  not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#if defined(TARGET_LIKE_K64F)
```

Comment:
I think we can, yes. Also, wouldn't it be enough to use `hal/targets/hal/TARGET_K64F` instead of `hal/targets/hal/TARGET_Freescale/TARGET_KSDK2_MCUS/TARGET_K64F` at this point?